src/main/java/com/alkemy/ong/security/JwtRequestFilter.java

```diff
@@ -0,0 +1,63 @@
+package com.alkemy.ong.security;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Component
+public class JwtRequestFilter extends OncePerRequestFilter {
+    @Autowired
+    private UserSecurityService userSecurityService;
+
+    @Autowired
+    private JwtUtils jwtUtils;
+
+    @Lazy
+    @Autowired
+    private AuthenticationManager authenticationManager;
+
+    @Value("${jwt.header}")
+    private String HEADER;
+
+    @Value("${jwt.tokenPrefix}")
+    private String TOKEN_PREFIX;
+
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws ServletException, IOException {
+        final String authorizationHeader = request.getHeader(this.HEADER);
+
+        String username = null;
+        String jwt = null;
+
```

Comment:
Could you please wrap the code below inside a try catch just in case. Like:

```java
try {
// all logic
} catch (Exception e) {
  logger.warn("Invalid jwt token exception due " + e);
}
chain.doFilter(request, response);
```